var/spack/repos/builtin/packages/py-simplejson/package.py

```diff
@@ -27,5 +28,5 @@ class PySimplejson(PythonPackage):
     version("3.8.0", sha256="217e4797da3a9a4a9fbe6722e0db98070b8443a88212d7acdbd241a7668141d9")
     version("3.3.0", sha256="7a8a6bd82e111976aeb06138316ab10847adf612925072eaff8512228bcf9a1f")
 
-    depends_on("python@2.5:2.8,3.3:", type=("build", "run"), when="@3.16.0:")
+    depends_on("python@2.5:2,3.3:", when="@3.16.0:", type=("build", "run"))
```

Comment:
You could honestly just delete this line, Spack only supports Python 3.7+ anyway

Reply:
I'm always reluctant to just delete working specs (just in case) :-)